compose/lite/configuration.yml

```diff
@@ -0,0 +1,91 @@
+###############################################################
+#                   Authelia configuration                    #
+###############################################################
+
+host: 0.0.0.0
+port: 9091
+
+logs_level: debug
+
+# This secret can also be set using the env variables AUTHELIA_JWT_SECRET
+jwt_secret: a_very_important_secret
+
+default_redirection_url: https://auth.example.com
+
+totp:
+  issuer: authelia.com
+
+duo_api:
+  hostname: api-123456789.example.com
+  integration_key: ABCDEF
+  # This secret can also be set using the env variables AUTHELIA_DUO_API_SECRET_KEY
+  secret_key: 1234567890abcdefghifjkl
+
+authentication_backend:
+  file:
+    path: /etc/authelia/users_database.yml
+
+access_control:
+  default_policy: deny
+
+  rules:
+    # Rules applied to everyone
+    - domain: auth.example.com
+      policy: bypass
+
+    - domain: traefik.example.com
+      policy: one_factor
+
+    - domain: secure.example.com
+      policy: two_factor
+
+session:
+  name: authelia_session
+
+  # This secret can also be set using the env variables AUTHELIA_SESSION_SECRET
+  secret: unsecure_session_secret
+
+  expiration: 3600 # 1 hour
+
+  inactivity: 300 # 5 minutes
+
+  domain: example.com # This should match whatever your root protected domain is
+
+  redis:
+    host: redis
+    port: 6379
+    # This secret can also be set using the env variables AUTHELIA_SESSION_REDIS_PASSWORD
+    password: authelia
+
+regulation:
+  max_retries: 3
+
+  find_time: 120
+
+  ban_time: 300
+
+storage:
+  local:
+    path: /var/lib/authelia/db.sqlite3
+
+notifier:
+  smtp:
```

Comment:
We might want to utilize the file based notifier for this example unless we're providing an SMTP server with it.

Reply:
I had originally considered this, my thought process was that the general consensus is that most people will have access to an mail server, most which are well documented as to how you would utilise these with your own mail clients/services and also the Gmail example which is partly why I included that commented.

As Clèment has suggested users that really want to customise can look at the config examples and make whatever determination that is necessary.

Reply:
Yeah that makes sense.